assets/js/print.mjs

```diff
@@ -0,0 +1,28 @@
+const initPrintLinks = () => {
+  /**
+   * Expand all the Details expander elements on print
+   */
+  const showDetailsElementOnPrint = () => {
+    document.querySelectorAll('.govuk-details').forEach(details => {
+      details.setAttribute('open', '')
+    })
+  }
+
+  window.addEventListener('beforeprint', showDetailsElementOnPrint)
+
+  /**
+   * Handle the print page link event
+   */
+  const printLink = document.getElementById('print-link')
```

Comment:
As and when we get stories to implement a "print this page" link, these functions should auto-magically step in to help us 👍